config/configmap/inferenceservice.yaml

```diff
@@ -82,6 +84,12 @@ data:
            # cpuLimit is the limits.cpu to set for the storage initializer init container.
            "cpuLimit": "1",
        
+           # caBundleSecretName is the Secret will be copied to a user namespace for the storage initializer init container.
+           "caBundleSecretName": "",
```

Comment:
I wonder if `ConfigMaps` should be used over `Secrets`.

Typically, CA Bundles do not contain private keys and these should not be need for verification purposes. So, I find more appropriate to use `ConfigMaps`. In case the CABundle of the user would contain any private keys,   that can represent a security risk and the recommendation should be to remove the keys from the bundle. Using a `ConfigMap` rather than a `Secret` would make this more evident to users.

Reply:
good catch! @yuzisun what do you think? if you agree, I will update this source but this is a little much changes though

Reply:
I convert the secret to configmap.